`make install` tries to install nonexistent file

Hi :)

installing 1.6 (make build-simavr && make install) fails:

```
Makefile:33: warning: overriding recipe for target 'install'
Makefile:27: warning: ignoring old recipe for target 'install'
make -C simavr install
make[1]: Entering directory '/build/simavr/src/simavr/simavr'
make obj config
make[2]: Entering directory '/build/simavr/src/simavr/simavr'
make[2]: Nothing to be done for 'obj'.
make[2]: Nothing to be done for 'config'.
make[2]: Leaving directory '/build/simavr/src/simavr/simavr'
make libsimavr run_avr
make[2]: Entering directory '/build/simavr/src/simavr/simavr'
make[2]: Nothing to be done for 'libsimavr'.
LD obj-x86_64-pc-linux-gnu/run_avr.elf
ln -sf obj-x86_64-pc-linux-gnu/run_avr.elf run_avr
make[2]: Leaving directory '/build/simavr/src/simavr/simavr'
mkdir -p /build/simavr/pkg/simavr/usr/include/simavr/avr
mkdir -p /build/simavr/pkg/simavr/usr/include/simavr/parts
install -m644 sim/*.h /build/simavr/pkg/simavr/usr/include/simavr/
install -m644 sim_core_*.h /build/simavr/pkg/simavr/usr/include/simavr/
install -m644 sim/avr/*.h /build/simavr/pkg/simavr/usr/include/simavr/avr/
install -m644 ../examples/parts/*.h /build/simavr/pkg/simavr/usr/include/simavr/parts/
mkdir -p /build/simavr/pkg/simavr/usr/lib
install obj-x86_64-pc-linux-gnu/libsimavr.a /build/simavr/pkg/simavr/usr/lib/
install ../examples/parts/obj-x86_64-pc-linux-gnu/libsimavrparts.a /build/simavr/pkg/simavr/usr/lib/
install: cannot stat '../examples/parts/obj-x86_64-pc-linux-gnu/libsimavrparts.a': No such file or directory
make[1]: *** [Makefile:89: install] Error 1
make[1]: Leaving directory '/build/simavr/src/simavr/simavr'
make: *** [Makefile:33: install] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -23,14 +23,11 @@ build-examples: build-simavr
 build-parts: build-examples
 	$(MAKE) -C examples/parts RELEASE=$(RELEASE)
 
-install:
+install: build-simavr build-parts
 	$(MAKE) -C simavr install RELEASE=$(RELEASE)
 
 doc:
 	$(MAKE) -C doc RELEASE=$(RELEASE)
-
-install:
-	$(MAKE) -C simavr install
 
 clean:
 	$(MAKE) -C simavr clean

--- a/Makefile
+++ b/Makefile
@@ -23,7 +23,7 @@ build-examples: build-simavr
 build-parts: build-examples
 	$(MAKE) -C examples/parts RELEASE=$(RELEASE)
 
-install: build-simavr build-parts
+install: build-parts
 	$(MAKE) -C simavr install RELEASE=$(RELEASE)
 
 doc:

